src/components/button-minor/button-minor.component.tsx

```diff
@@ -30,11 +31,15 @@ export const ButtonMinor = React.forwardRef<
       iconPosition: iconPositionContext,
       fullWidth: fullWidthContext,
     } = useContext(ButtonBarContext);
+    const { disabled: batchSelectionDisabled } = useContext(
+      BatchSelectionContext
+    );
 
     const buttonType = buttonTypeContext || buttonTypeProp;
     const size = sizeContext || sizeProp;
     const iconPosition = iconPositionContext || iconPositionProp;
     const fullWidth = fullWidthContext || fullWidthProp;
+    const isDisabled = disabled || batchSelectionDisabled || false;
```

Comment:
```suggestion
    const isDisabled = disabled || batchSelectionDisabled;
```

If we keep both the prop and context default values we don't need the last part here